tendrl/monitoring_integration/alert/utils.py

```diff
@@ -214,3 +214,20 @@ def find_volume_name(integration_id, hostname, brick_path):
             }
         )
         raise ex
+
+
+def find_cluster_short_name(integration_id):
+    try:
+        cluster = NS.tendrl.objects.Cluster(
+            integration_id=integration_id
+        ).load()
+        return cluster.short_name
```

Comment:
shouldnt you `return cluster.short_name or cluster.integration_id`  

@GowthamShanmugam @nthomas-redhat

Reply:
I thought if the username is not given by user then backed is assigning integration_id as  short_name in cluster object. I will change this